frontend/src/v5/ui/controls/errorMessage/networkError/networkError.component.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { formatMessage } from '@/v5/services/intl';
+import { ErrorMessage } from '../errorMessage.component';
+
+export const NETWORK_ERROR_MESSAGE = formatMessage({
+	id: 'editProfile.networkError',
+	defaultMessage: 'Network Error',
+});
+
+export const NetworkError = () => <ErrorMessage>{NETWORK_ERROR_MESSAGE}</ErrorMessage>;
```

Comment:
Nice! I think this component is a good idea. 

I would use all React components instead of the format message bit.  Also this way the component is more encapsulated so you would be forced to not use the constant in the other components.


```suggestion
export const NetworkError = () => (
	<ErrorMessage>
		<FormattedMessage id="editProfile.networkError" defaultMessage="Network Error" />
	</ErrorMessage>
);

```